frame/contracts/src/wasm/code_cache.rs

```diff
@@ -218,14 +218,16 @@ impl<T: Config> Token<T> for CodeToken {
 		// contract code. This is why we subtract `T::*::(0)`. We need to do this at this
 		// point because when charging the general weight for calling the contract we not know the
 		// size of the contract.
-		match *self {
+		let ref_time_weight = match *self {
 			Reinstrument(len) => T::WeightInfo::reinstrument(len),
 			Load(len) => {
 				let computation = T::WeightInfo::call_with_code_per_byte(len)
 					.saturating_sub(T::WeightInfo::call_with_code_per_byte(0));
-				let bandwidth = T::ContractAccessWeight::get().saturating_mul(len.into());
+				let bandwidth = T::ContractAccessWeight::get().scalar_saturating_mul(len as u64);
```

Comment:
Otherwise we wouldn't pick this up when refactoring `RefTimeWeight`.

```suggestion
				let bandwidth = T::ContractAccessWeight::get().scalar_saturating_mul(len as RefTimeWeight);
```

Reply:
i think i will just remove reftimeweight for u64 directly anyway